lib/src/l10n/localizations.dart

```diff
@@ -93,6 +93,46 @@ class AppLocalizations {
   String get camera => Intl.message('Camera', name: 'camera');
 
+  // Core
+  String get coreChatNoMessages => Intl.message('No messages', name: 'coreChatNoMessages');
+  String get coreSelf => Intl.message('Me', name: 'coreSelf');
+  String get coreDraft => Intl.message('Draft', name: 'coreDraft');
+  String get coreMembers => Intl.message('%1\$d member(s)', name: 'createChatWith');
+  String get coreContacts => Intl.message('%1\$d contact(s)', name: 'createChatWith');
+  String get coreVoiceMessage => Intl.message('Voice message', name: 'coreVoiceMessage');
+  String get coreContactRequest => Intl.message('Contact request', name: 'coreContactRequest');
+  String get coreImage => Intl.message('Image', name: 'coreImage');
+  String get coreVideo => Intl.message('Video', name: 'coreVideo');
+  String get coreAudio => Intl.message('Audio', name: 'coreAudio');
+  String get coreFile => Intl.message('File', name: 'coreFile');
+  String get coreGroupHelloDraft => Intl.message('Hello, I\'ve just created this group for us', name: 'coreGroupHelloDraft');
+  String get coreGroupNameChanged => Intl.message('Group name changed', name: 'coreGroupNameChanged');
+  String get coreGroupImageChanged => Intl.message('Group image changed', name: 'coreGroupImageChanged');
+  String get coreGroupMemberAdded => Intl.message('Member added', name: 'coreGroupMemberAdded');
+  String get coreGroupMemberRemoved => Intl.message('Member removed', name: 'coreGroupMemberRemoved');
+  String get coreGroupLeft => Intl.message('Group left', name: 'coreGroupLeft');
+  String get coreGenericError => Intl.message('Error: ', name: 'coreGenericError');
+  String get coreGif => Intl.message('Gif', name: 'coreGif');
+  String get coreMessageCannotDecrypt => Intl.message('This message cannot be decrypted.\n\n'
+      '• It might already help to simply reply to this message and ask the sender to send the message again.\n\n'
+      '• In case you re-installed Delta Chat or another email program on this or another device you may want to send an Autocrypt Setup Message from there.', name: 'coreMessageCannotDecrypt');
+  String get coreReadReceiptSubject => Intl.message('Read receipt', name: 'coreReadReceiptSubject');
+  String get coreReadReceiptBody => Intl.message('This is a read receipt.\n\nIt means the message was displayed on the recipient\'s device, not necessarily that the content was read.', name: 'coreReadReceiptBody');
+  String get coreGroupImageDeleted => Intl.message('Group image deleted', name: 'coreGroupImageDeleted');
+  String get coreContactVerified => Intl.message('Contact verified', name: 'coreContactVerified');
+  String get coreContactNotVerified => Intl.message('Cannot verify contact', name: 'coreContactNotVerified');
+  String get coreContactSetupChanged => Intl.message('Changed setup for contact', name: 'coreContactSetupChanged');
+  String get coreArchivedChats => Intl.message('aaaaaaaaaaaaaaaa', name: 'coreArchivedChats');
```

Comment:
Is 'aaaaaaaaaaaaaaaa' the real string?

Reply:
Fixed by https://github.com/open-xchange/ox-talk/pull/23/commits/bf32e663a17bfe33a37f3dbf8d6c2556dcfd11a9